taskcluster/tc-python_tflite-tests-prod.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -xe
+
+source $(dirname "$0")/tc-tests-utils.sh
+
+extract_python_versions "$1" "pyver" "pyver_pkg" "py_unicode_type" "pyconf" "pyalias"
+
+unset PYTHON_BIN_PATH
+unset PYTHONPATH
+
+if [ -d "${DS_ROOT_TASK}/pyenv.cache/" ]; then
+  export PYENV_ROOT="${DS_ROOT_TASK}/pyenv.cache/ds-test/.pyenv"
+else
+  export PYENV_ROOT="${DS_ROOT_TASK}/ds-test/.pyenv"
+fi;
+
+export PATH="${PYENV_ROOT}/bin:$PATH"
+
+mkdir -p ${PYENV_ROOT} || true
+
+model_source=${DEEPSPEECH_PROD_MODEL//.pb/.tflite}
```

Comment:
This needs a similar override for the URL as cpp-tflite-tests right?

Reply:
It would, but right now we don't run Python prod tests at all. Should we ? I think we should.